unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/TaskbarHUDView.cs

```diff
@@ -65,6 +72,12 @@ internal List<TaskbarButton> GetButtonList()
         taskbarButtonList.Add(settingsButton);
         taskbarButtonList.Add(exploreButton);
         taskbarButtonList.Add(moreButton);
+
+        foreach (var portableExperience in activePortableExperienceItems)
+        {
+            taskbarButtonList.Add(portableExperience.Value.mainButton);
+        }
```

Comment:
same as before. I think we decided to try to avoid the use of the `foreach`

Reply:
In this case, the entity to iterate is a `Dictionary` and it seems the more performant way to iterate them is using the `foreach`. I was researched a bit and I found this article: https://social.technet.microsoft.com/wiki/contents/articles/48007.c-different-ways-of-iterating-over-a-dictionary-and-their-performance.aspx

Reply:
Yeah I know that `foreach` in lot of cases has better performance, but I think the concerns where about allocations. I don't know if that currently applies. We should probably discuss this all together to reach some consensus cause I will love if we can start using `foreach` :D